spdx-utils/src/main/kotlin/SpdxLicenseStringMapping.kt

```diff
@@ -25,7 +25,7 @@ import com.here.ort.spdx.SpdxLicenseException.*
 /**
  * A mapping from license strings collected from the declared licenses of open source packages to SPDX expressions.
  */
-object SpdxLicenseMapping {
+object SpdxLicenseStringMapping {
```

Comment:
Commit message typo in "SdpxLicenseIdMapping".

Reply:
Done.